web/src/features/AppVersionHistory/AppVersionHistoryRow.tsx

```diff
@@ -180,12 +186,12 @@ function AppVersionHistoryRow(props: Props) {
   };
 
   const renderVersionAction = (version: Version) => {
-    const app = props.app;
+    const app = selectedApp;
     const downstream = app?.downstream;
```

Comment:
Can this be destructured from selectedApp? 
```suggestion
const { downstream } = selectedApp
```